source/embedded-shell.txt

```diff
@@ -12,24 +12,24 @@ Embedded MongoDB Shell
    :depth: 1
    :class: singlecol
 
-Starting in version 1.22, |compass| contains an embedded
-`MongoDB Shell <https://docs.mongodb.com/mongodb-shell/>`__. The
-MongoDB Shell is a fully functional JavaScript environment for
-interacting with MongoDB deployments. You can use the MongoDB Shell to
-test queries and operations directly with your database.
+Starting in version 1.22, |compass| contains an embedded shell,
+:binary:`mongosh`. :binary:`mongosh` is a fully functional
+JavaScript environment for interacting with MongoDB deployments. You
+can use :binary:`mongosh` to test queries and operations directly
+with your database.
 
-Open the Embedded MongoDB Shell
--------------------------------
+Open the Embedded ``mongosh``
```

Comment:
This might be okay, but it just reads a little odd. But, there's a lot of controversy on how our products are referred to like this. If it is decided we have to write it like this, then so be it.

But, maybe keep the original heading "Open the Embedded MongoDB Shell", which reads better IMO. Same comment for other headings.